example_site/src/components/App.css

```diff
@@ -113,3 +108,7 @@ div.loading {
   width: 150px;
   margin-left: 20px;
 }
+
+.move-up {
+  margin-top: -25px;
+}
```

Comment:
We should not need this (although it's a clever solution!).

The div this is applied to also has another class, `panel`. `panel` has 24px of padding applied to it, so that explains the -25px margin here. The `panel` class is leftover from the old site layout, so I would get rid of it here, or look around at other places its being applied to see if it's really relevant anymore.